### ISSUE TITLE
some mistake in the code

### ISSUE THREAD

**Issue by tobelyf** [jeong-tae/RACNN-pytorch#22]
After line 183 in the trainer.py , did you forget this line of code "logits, _, _ = net(images)"?

**Comment by jeong-tae** [comment]
OMG, I guess yes! you are right. Thank you!
It may not effect to train but it makes hard to understand progress of learning. thank you!

https://github.com/jeong-tae/RACNN-pytorch/commit/3301753e660a8453d69b918d87950ff29a3c96ea applied in this commit
